io/disk/htx_block_devices.py

```diff
@@ -118,8 +118,10 @@ def test(self):
         self.log.info("Running the HTX on %s", self.block_device)
         cmd = "htxcmdline -run %s -mdt %s" % (self.block_device, self.mdt_file)
         process.system(cmd, ignore_status=True)
-        for _ in range(0, self.time_limit, 60):
-            self.run_smt()
+        for time_loop in range(0, self.time_limit, 60):
+            # Running SMT changes every hour
+            if time_loop % 3600 == 0:
```

Comment:
i am not sure after a hour smt change will really effect the test so i suggested can we get some approx time of SMT change may be 5 min should be ok , what you think